library.properties

```diff
@@ -9,4 +9,4 @@ url=https://github.com/leaphy-robotics/leaphy-extensions-extra
 architectures=avr
 # Only include actually used library code in the resulting executable
 dot_a_linkage=true
-depends=Adafruit SSD1306,Adafruit GFX Library,Adafruit LSM303 Accel,Adafruit LIS2MDL,Adafruit BusIO,Adafruit_VL53L0X,Adafruit NeoPixel
+depends=Adafruit SSD1306,Adafruit GFX Library,Adafruit LSM303 Accel,Adafruit LIS2MDL,Adafruit BusIO,Adafruit_VL53L0X,Adafruit NeoPixel,List
```

Comment:
vergeet niet om de version te updaten, anders komt hij niet in de arduino lib index